index.js

```diff
@@ -101,6 +104,9 @@ module.exports = {
   },
 
   serverMiddleware: function ({ app: expressApp, options }) {
+    if (process.env.DISABLE_CSP) {
```

Comment:
Any reason to have an environment variable to disable CSP in addon hard-coded? Having it in userland configuration would reduce complexity. A consumer would not need to look into addons documentation but only in project's CSP configuration to discover and understand the environment variable.

Reply:
this allows libraries to easily turn it off via config when running the command vs needing to learn about and change config files